.github/ISSUE_TEMPLATE/3_product_release_checks.md

```diff
@@ -33,9 +26,24 @@ Make sure to open and fill in a separate **documentation issue** in your product
 > [!NOTE]
 > Note: most criteria for documentation and security are now covered in TRGs
 
-# Security Checks
+# Features
```

Comment:
Please add empty line for proper md format

Reply:
done :)